src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -684,7 +691,7 @@ void GcsPlacementGroupManager::Initialize(const GcsInitData &gcs_init_data) {
   // Notify raylets to release unused bundles.
   gcs_placement_group_scheduler_->ReleaseUnusedBundles(node_to_bundles);
 
-  SchedulePendingPlacementGroups();
+  RetryCreatingPlacementGroup(0);
```

Comment:
This is unnecessary. I will remove it.

Reply:
I think it will actually have positive impact (since the consecutive schedule function won't block things too long in this way), so I will keep it until #18651 is resolved. After that, it should be removed naturally.